kagami/webui/src/api/resourceInfo.ts

```diff
@@ -0,0 +1,27 @@
+import api from './api';
+
+export interface ResourceInfo {
+    name:string;
+    status:string;
+    helper:boolean;
+    };
+
+
+export const getResource = async (): Promise<ResourceInfo[]> => {
+    try {
+      const response = await api.get('/resource_info');
+      return response.data;
+    } catch (error) {
+      console.error('获取公告失败:', error);
+      throw error;
+    }
+  };
+  export const createAnnouncement = async (resource: ResourceInfo): Promise<ResourceInfo[]> => {
+    try {
+      const response = await api.post('/resource', resource);
```

Comment:
api.post<ResourceInfo[]>('/resource', resource);

You can add type to response data like this.